src/v2/view-builder/views/consent/AdminConsentViewHeader.js

```diff
@@ -12,7 +12,7 @@ const AdminConsentViewHeader = View.extend({
     <a href="{{clientURI}}" class="client-logo-link" target="_blank">
     {{/if}}
     {{#if customLogo}}
-      <img class="client-logo custom-logo" src="{{customLogo}}" alt="{{altText}}" aria-hidden="true" />
+      <img class="client-logo custom-app-logo" src="{{customLogo}}" alt="{{altText}}" aria-hidden="true" />
```

Comment:
I don't think you need to change this file, this is for a different view

Reply:
Yes Good catch.. i thought i reverted this but somehow it slipped.